app/presenters/admin/purchase_presenter.rb

```diff
@@ -29,4 +32,105 @@ def list_props
       last_chargebacked_purchase: purchase.find_past_chargebacked_purchases.first&.id,
     }
   end
+
+  def props
+    base_props = list_props
+    email_info = CustomerEmailInfo.where(purchase: purchase, email_name: purchase.is_preorder_authorization ? "preorder_receipt" : "receipt").last
+    email_info_text = email_info.try(:state).try(:capitalize)
+    email_info_text += " (on #{email_info.delivered_at})" if email_info.try(:delivered?)
+    email_info_text += " (on #{email_info.opened_at})" if email_info.try(:opened?)
+    base_props[:refund_policy]&.merge!({
+                                         fine_print: purchase.purchase_refund_policy.fine_print.presence || purchase.link.product_refund_policy&.fine_print,
+                                         max_refund_period_in_days: purchase.purchase_refund_policy.max_refund_period_in_days
+                                       })
+    base_props.merge({
+                       updated_at: purchase.updated_at,
```

Comment:
Nit: Might be nice to use `reverse_merge` or otherwise change this to avoid this entire hash having so much indentation

Reply:
Personally I think this alignment is super weird to begin with - we changed this in Flexile a while back (https://github.com/antiwork/flexile-old/pull/2829) and I think we should do the same thing here. Wdyt? cc @slavingia

Reply:
I'd be super keen to change it!